server/lib/nl/common/utterance.py

```diff
@@ -93,10 +93,22 @@ class ChartType(IntEnum):
 
 class FulfillmentResult(str, Enum):
   """Where is a variable or place from?"""
+  # From the current user provided query.
   CURRENT_QUERY = "CURRENT_QUERY"
+  # From a past user query in the context.
+  # Importantly, no place/variable was recognized in the current query.
```

Comment:
What does this comment mean? Is ``PAST_QUERY`` only set if no place/variable was recognized int he current query?

Reply:
Correct.